signer/src/message.rs

```diff
@@ -322,6 +335,47 @@ pub struct BitcoinTransactionSignAck {
     pub txid: bitcoin::Txid,
 }
 
+/// The message version of an the [`OutPoint`].
+#[derive(Debug, Copy, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct OutPointMessage {
```

Comment:
I'd rename this to `BitcoinOutPoint`.

Reply:
Actually, we can keep it as is. When we switch to protobufs we can remove this and use the `proto::OutPoint` type.